bok_choy/page_object.py

```diff
@@ -500,3 +520,130 @@ def wait_for_element_invisibility(self, element_selector, description, timeout=6
 
         """
         self.wait_for(lambda: not self.q(css=element_selector).visible, description=description, timeout=timeout)
+
+    def axs_audit_rules(self):
```

Comment:
Not a must, but would it make sense to have these in a different class that gets inherited by PageObject?